modules/auxiliary/fileformat/badpdf.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::FILEFORMAT
+
+  def initialize(info={})
+    super( update_info( info,
+        'Name'          => 'BADPDF Malicious PDF Creator',
+        'Description'   => %q{
+          This module can either creates a blank PDF file which contains a UNC link which can be used
+          to capture NetNTLM credentials, or if the PDFINJECT option is used it will inject the necessary
+          code into an existing PDF document if possible.
+        },
+        'License'       => MSF_LICENSE,
+        'Author'        =>
+            [
+              'Richard Davy - secureyourit.co.uk',  #Module written by Richard Davy
+              'CheckPoint researchers - Assaf Baharav, Yaron Fruchtmann, Ido Solomon' #Code provided as POC by CheckPoint Researchers
```

Comment:
Usually authors are split into one per each `Author` array element. Something like this:

```ruby
'Author' => [
  'Assaf Baharav',    # Code provided as POC by CheckPoint 
  'Yaron Fruchtmann', # Code provided as POC by CheckPoint 
  'Ido Solomon',      # Code provided as POC by CheckPoint 
  'Richard Davy - secureyourit.co.uk',  # Metasploit
]
```